src/components/common/YourCompany/CorrectName/CorrectNameToNumber.vue

```diff
@@ -9,7 +9,7 @@
           id="correct-name-to-number-checkbox"
           v-model="correctToNumbered"
           class="mb-n5"
-          :label="`Change the company name to ${businessId} B.C. Ltd.`"
+          :label="`Change the company name to ${businessId} B.C. ${getUpdatedName()}`"
         />
```

Comment:
fixed the error: change Named business to Numbered -> showing wrong numbered name, always showed "B.C. Ltd".